django/django-quiz.md

```diff
@@ -1,333 +1,333 @@
-Q1. To cache your entire site for an application in Django, you add all except which of these settings?
+#### Q1. To cache your entire site for an application in Django, you add all except which of these settings?
 
-    - django.middleware.common.CommonMiddleware
-    - django.middleware.cache.UpdateCacheMiddleware
-    - django.middleware.cache.FetchFromCacheMiddleware
-    👍 - django.middleware.cache.AcceleratedCacheMiddleware
+    - [] django.middleware.common.CommonMiddleware
+    - [] django.middleware.cache.UpdateCacheMiddleware
+    - [] django.middleware.cache.FetchFromCacheMiddleware
+    - [x] - django.middleware.cache.AcceleratedCacheMiddleware
 
-Q2. In which programming language is Django written?
+#### Q2. In which programming language is Django written?
 
-    - C++
-    - Java
-    👍 - Python
-    - Ruby
+    - [] C++
+    - [] Java
+    - [x] - Python
+    - [] Ruby
 
-Q3. To automatically provide a value for a field, or to do validation that requires access to more than a single field, you should override the**\_** method in the**\_** class.
+#### Q3. To automatically provide a value for a field, or to do validation that requires access to more than a single field, you should override the**\_** method in the**\_** class.
 
-    - validate(); Model
-    - group(); Model
-    - validate(); Form
-    👍 - clean(); Field
+    - [] validate(); Model
+    - [] group(); Model
+    - [] validate(); Form
+    - [x] - clean(); Field
 
-Q4. A client wants their site to be able to load "Rick & Morty" episodes by number or by title—e.g., shows/3/3 or shows/picklerick. Which URL pattern do you recommend?
+#### Q4. A client wants their site to be able to load "Rick & Morty" episodes by number or by titleÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¿ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â½e.g., shows/3/3 or shows/picklerick. Which URL pattern do you recommend?
```

Comment:
please double-check why there are special characters instead of alphabetical symbols